packages/frontend/src/components/nft/NFTDetail.js

```diff
@@ -0,0 +1,225 @@
+
+    .owner {
+        p {
+            font-family: Inter;
+            font-style: normal;
+            font-weight: 500;
+            font-size: 12px;
+            line-height: 150%;
+            /* identical to box height, or 18px */
+
+            display: flex;
+            align-items: center;
+            letter-spacing: 0.115em;
+
+            /* gray/neutral/500 */
+
+            color: #A2A2A8;
+        }
+
+        .inner {
+            height: 30px;          
+            line-height: 30px; 
+            display: flex;         
+            align-items: center;
+
+            span {
+                font-family: Inter;
+                font-style: normal;
+                font-weight: 500;
+                font-size: 16px;
+                line-height: 150%;
+                /* identical to box height, or 24px */
+
+                display: flex;
+                align-items: center;
+
+                /* gray/neutral/800 */
+
+                color: #272729;
+            }
+        }
+    }
+
+    .transfer-btn {
+        width: 100%;
+        margin-top: 56px !important;
+
+        svg {
+            margin-right: 10px !important;
+        }
+    }
+
+    #back-btn {
+        position: absolute;
+        left: -98px;
+        width: 30px !important;
+        height: 30px !important;
+        
+        svg {
+            width: 100%;
+            height: 100%;
+        }
+    }
+
+`;
+
+const UserIcon = styled.div`
+    background-size: 21px;
+    flex: 0 0 40px;
+    height: 40px;
+    border-radius: 50%;
+    background-color: #f8f8f8;
+    text-align: center;
+    margin: 0 12px 0 0;
+    
+    svg {
+        width: 26px;
+        height: 26px;
+        margin: 7px;
+    }
+
+    @media (min-width: 940px) {
+        display: inline-block;
+    }
+`;
+
+function arrowSVG () {
+  return (
+    <svg className="transfer-svg" width="22" height="22" viewBox="0 0 22 22" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path d="M21 1L14 21L10 12L1 8L21 1Z" stroke="#A2A2A8" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round"/>
+      <path d="M21 1L10 12" stroke="#A2A2A8" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round"/>
+    </svg>
+  );
+}
+
+export function NFTDetail({ match, location, history }) {
+    const accountId = useSelector(selectAccountId);
+    const fungibleTokens = useFungibleTokensIncludingNEAR();
+    const nearBalance = fungibleTokens[0].balance;
+
+    const contractId = match.params.contractId;
+    const tokenId = match.params.tokenId;
+    const [ nft, setNft ] = useState();
+    const [ ownerId, setOwnerId ] = useState();
+    const [transferNftDetail, setTransferNftDetail] = useState();
+
+    useEffect(() => {
+        NonFungibleTokens.getToken(contractId, tokenId)
+            .then(token => {
+                token.contract_id = contractId;
+                setNft(token);
+                setOwnerId(token.owner_id);
+            });
+    }, []);
+
+    function getNEARBalance () {
```

Comment:
Is there a reason why we need a function to return `nearBalance` vs referencing the variable directly?